Updates docs for firmware changes for CM5 / Pi500 and other recent firmware changes. (#3935)
raspberrypi/documentation: documentation/asciidoc/computers/raspberry-pi/eeprom-bootloader.adoc

If `1` then `sudo halt` will run in a lower power mode until either GPIO3 or GLOBAL_EN are shorted to ground.

> This setting is not relevant on Raspberry Pi 5, CM5 or Pi500 because the xref:raspberry-pi.adoc#power-button[dedicated power button] may always be used to wakeup from `HALT` or `STANDBY`.

Reviewer: ```suggestion
This setting is not relevant on Raspberry Pi 5, Compute Module 5, or Raspberry Pi 500 because the xref:raspberry-pi.adoc#power-button[dedicated power button] may always be used to wake from `HALT` or `STANDBY`.
```

Minor improvements to wording.

> This setting is not relevant on Flagship models since 5, Compute Modules since 5, and Keyboard models since 500 because the xref:raspberry-pi.adoc#power-button[dedicated power button] may always be used to wake from `HALT` or `STANDBY`.

Reviewer: ```suggestion
This setting is not relevant on Flagship models since Raspberry Pi 5, Compute Modules since 5, and Keyboard models since 500 because the xref:raspberry-pi.adoc#power-button[dedicated power button] may always be used to wake from `HALT` or `STANDBY`.
```

On Raspberry Pi 5 this places the PMIC in `STANDBY` mode where all outputs are switched off. There is no need to set `WAKE_ON_GPIO` and pressing the xref:raspberry-pi.adoc#power-button[dedicated power button] will boot the device.

> Default: `1` on CM5 and Pi500, otherwise `0`

Reviewer: ```suggestion
Default: `1` on Compute Module 5 and Pi 500, otherwise `0`
```

> Default: `1` on Compute Module 5 and Pi 500, otherwise `0`

Reviewer: ```suggestion
Default: `1` on flagship models since Raspberry Pi 5, Compute Modules since Compute Module 5, and Keyboard models since Raspberry Pi 500, otherwise `0`
```

> Default: `1` on Compute Modules since 5 and Keyboard models since 400; otherwise `0`

Reviewer: ```suggestion
Default: `1` on Compute Modules since CM5 and Keyboard models since Pi 400; otherwise `0`
```

> Default: `1` on Compute Modules since Compute Module 5 and Keyboard models since Raspberry Pi 500; otherwise `0`

Reviewer: ```suggestion
Default: `1` on Compute Modules since 5 and Keyboard models since 500; otherwise `0`
```

Reviewer: @timg236 since 400 has a power button, I assume the keyboard models adopted this default in the 400 model.

In order to detect the keyboard, network install must initialise the USB controller and enumerate devices. This increases boot time by approximately 1 second so it may be advantageous to disable network install in some embedded applications.

Default: `1` on Raspberry Pi 4 and Raspberry Pi 400; `0` on Compute Module 4, Compute Module 4S, and Compute Module 5.

[[NET_INSTALL_AT_POWER_ON]]

> ==== `NET_INSTALL_AT_POWER_ON`
> If this property is set to `1` then the network install UI will briefly be displayed after a cold boot to make this feature more obvious to new users.
> The default bootloader images set this value to `1` in the bootloader config. It can be disabled via the `Advanced Options` menu in `raspi-config` or by manually deleting the line via `sudo rpi-eeprom-config --edit`

Reviewer: ```suggestion
==== `NET_INSTALL_AT_POWER_ON`

When set to `1`, displays the network install UI briefly after a cold boot to make this feature more obvious to new users.

The default bootloader images set this value to `1` in the bootloader config. To disable the brief network install UI display, use the `Advanced Options` menu in `raspi-config` or manually delete this line in `rpi-eeprom-config`:

[source,console]
----
$ sudo rpi-eeprom-config --edit
----

```

In order to detect the keyboard, network install must initialise the USB controller and enumerate devices. This increases boot time by approximately 1 second so it may be advantageous to disable network install in some embedded applications.

> Default: `1` on Raspberry Pi 4 and Raspberry Pi 400; `0` on Compute Module 4, Compute Module 4S, and Compute Module 5.

Reviewer 2: What does it default to on Pi 5?

Reviewer: ```suggestion
Default: `1` on Raspberry Pi 5, Raspberry Pi 4 and Raspberry Pi 400; `0` on Compute Module 4, Compute Module 4S, and Compute Module 5.
```

Reviewer 2: Presumably also `1` on Raspberry Pi 500 ? (since we're mentioning that anyway in this PR!)

> Default: `1` on Flagship models since 4 and Keyboard models since 400; `0` on Compute Modules since 4 (including Compute Module 4S).

Reviewer: ```suggestion
Default: `1` on Flagship models since Raspberry Pi 4 and Keyboard models since 400; `0` on Compute Modules since 4 (including Compute Module 4S).
```

> Default: `1` on Raspberry Pi 5, Raspberry Pi 4 and Raspberry Pi 400; `0` on Compute Module 4, Compute Module 4S, and Compute Module 5.

Reviewer: ```suggestion
Default: `1` on Raspberry Pi 5, Raspberry Pi 4, Raspberry Pi 500, and Raspberry Pi 400; `0` on Compute Module 4, Compute Module 4S, and Compute Module 5.
```

> Default: `1` on Raspberry Pi 5, Raspberry Pi 4, Raspberry Pi 500, and Raspberry Pi 400; `0` on Compute Module 4, Compute Module 4S, and Compute Module 5.

Reviewer: ```suggestion
Default: `1` on Flagship models since 4 and Keyboard models since 400; `0` on Compute Modules since 4 and Compute Module 4S.
```

[[NET_INSTALL_AT_POWER_ON]]
==== `NET_INSTALL_AT_POWER_ON`

> When set to `1`, displays the network install UI briefly after a cold boot to make this feature more obvious to new users.

Reviewer 2: Presumably this setting is ignored if `NET_INSTALL_ENABLED=0` ? (is that worth mentioning here?)

Reviewer 2: Ah! That's probably worth mentioning too :slightly_smiling_face:

Author: That's interesting, I thought it was the other way around although I think I prefer NET_INSTALL_AT_POWER_ON=1 overriding NET_INSTALL_ENABLED

Reviewer: ```suggestion
When set to `1`, displays the network install UI briefly after a cold boot to make this feature more obvious to new users. Overrides `NET_INSTALL_ENABLED` if the settings conflict.
```
